osu.Game/Overlays/Options/General/LoginOptions.cs

```diff
@@ -10,38 +11,72 @@
 
 namespace osu.Game.Overlays.Options.General
 {
-    public class LoginOptions : OptionsSubsection
+    public class LoginOptions : OptionsSubsection, IOnlineComponent
```

Comment:
not too sure that's something to worry about. is an unfortunate limitation of interfaces,